crates/re_renderer/src/view_builder.rs

```diff
@@ -687,11 +712,54 @@ impl ViewBuilder {
         Ok(ScheduledScreenshot {
             row_info,
             identifier,
-            width: setup.resolution_in_pixel[0],
-            height: setup.resolution_in_pixel[1],
+            extent: setup.resolution_in_pixel.into(),
         })
     }
 
+    /// Schedules the readback of a rectangle from the picking layer.
+    ///
+    /// The result will still be valid if the rectangle is partially or fully outside of bounds.
+    /// Areas that are not overlapping with the primary target will be filled as-if the view's target was bigger.
```

Comment:
filled with what? zeroes?

Reply:
filled with content as if the view's target was bigger
will try to clarify, not exactly sure tho how